ci/build-unix.yml

```diff
@@ -34,11 +34,23 @@ steps:
       # infra/macos/2-common-box/init.sh:echo "build:darwin --disk_cache=~/.bazel-cache" > ~/.bazelrc
       # infra/vsts_agent_linux_startup.sh:echo "build:linux --disk_cache=~/.bazel-cache" > ~/.bazelrc
       disk_cache=$HOME/.bazel-cache
-      if [ $(du -ms $disk_cache | awk '{print $1}') -gt 80000 ]; then
-          echo "Deleting $disk_cache"
-          rm -rf $disk_cache
+      if [ $(du -ms "$disk_cache" | awk '{print $1}') -gt 80000 ]; then
+          echo "Deleting '$disk_cache'..."
+          rm -rf "$disk_cache"
       else
-          echo "Disk cache small enough: $(du -hs $disk_cache)."
+          echo "Disk cache small enough:\n$(du -hs "$disk_cache")"
+      fi
+      local_cache=$HOME/.cache/bazel
+      if [ -d "$local_cache" ]; then
+          if [ $(du -ms "$local_cache" | awk '{print $1}') -gt 80000 ]; then
+              echo "Deleting '$local_cache'..."
+              find "$local_cache" -type d | xargs chmod +w
```

Comment:
Shouldn’t the `-f` be sufficient to not require a `chmod +w`?

Reply:
I expected that too, but my testing says otherwise 🤷 

```
$ mkdir protected
$ touch protected/file
$ chmod -w protected
$ rm -f protected/file
rm: cannot remove 'protected/file': Permission denied
$ uname -a
Linux ubuntu-bionic 4.15.0-130-generic #134-Ubuntu SMP Tue Jan 5 20:46:26 UTC 2021 x86_64 x86_64 x86_64 GNU/Linux
$ 
```

Reply:
I admit I’m very confused now but I can reproduce so :shrug: